docs-website/docs/docs/Components.md

```diff
@@ -93,6 +93,11 @@ const EnhancedComment = enhance(Comment)
 
 **Note** again that we can also pass `Relation` objects directly for convenience, skipping `.observe()`
 
+### Reactive optional relations
+
+Continuing the above example, suppose the comment author is optional. If the comment has no author, the `comment.author_id` must be null. If `comment.author_id` has a value, the author record it refers to must be stored in the database, otherwise `withObservables` will throw an error that the record was not found.
```

Comment:
```suggestion
Continuing the above example, if the comment has no author, the `comment.author_id` must be null. If `comment.author_id` has a value, the author record it refers to must be stored in the database, otherwise `withObservables` will throw an error that the record was not found.
```